docker/maintenance/update-fandom-shop-cache.yaml

```diff
@@ -0,0 +1,4 @@
+schedule: @hourly
+args:
+- php
+- /usr/wikia/slot1/current/src/maintenance/wikia/cronjobs/updateFandomShopCache.php
```

Comment:
Is there going to be a similar job in UCP? Or are we only adding it to the legacy app?

Reply:
There will need to be one in UCP too.

Reply:
The plan is to have similar job in UCP once this is complete.

https://github.com/Wikia/unified-platform/pull/3235